core/server/api/v2/utils/serializers/output/utils/members.js

```diff
@@ -0,0 +1,14 @@
+const labsUtil = require('../../../../../../services/labs');
+
+const forPost = (attrs, frame) => {
+    const hideMemberOnlyContent = labsUtil.isSet('members') && !frame.context.member;
+    if (hideMemberOnlyContent) {
+        ['plaintext', 'html'].forEach((field) => {
+            attrs[field] = '';
```

Comment:
shouldn't we show some kind of warning or just completely remove the post instead?

Reply:
The warning will be handled in the templates for now we think.
e.g.
```hbs
{{#if hasContent}}
  {{content}}
{{else}}
  become a member yo'
{{/if}}
```